.env.example

```diff
@@ -16,6 +16,8 @@ LOCAL_CIRCLE_ADMIN=0xYOUR_DEV_ADDRESS
 # Local Environment
 REACT_APP_API_BASE_URL=http://localhost:9000/api
 REACT_APP_HASURA_URL=http://localhost:8080/v1/graphql
+# local version of s3 for local testing
+REACT_APP_S3_BASE_URL=http://coordinape.s3-website.localhost.localstack.cloud:4566
```

Comment:
I wonder if this can use a URL, that is just `http://localhost:4566` otherwise not a big deal.